src/emulator/commandUtils.ts

```diff
@@ -46,81 +46,60 @@ export const DESC_TEST_PARAMS =
  */
 const DEFAULT_CONFIG = new Config({ database: {}, firestore: {}, functions: {}, hosting: {} }, {});
 
-export function warnRealtimeDatabaseEmulated(
-  options: any,
-  emulatorSupported: boolean
-): void | Promise<void> {
-  const envKey = Constants.FIREBASE_DATABASE_EMULATOR_HOST;
+export function printNoticeIfEmulated(options: any, emulator: Emulators): void {
```

Comment:
```suggestion
export function printNoticeIfEmulated(options: any, emulator: Emulators.DATABASE | Emulators.FIRESTORE): void {
```

Right now this function only supports those two emulators so let's just be frank about it. Right now, you can pass in `Emulators.HOSTING` and it actually checks the Firestore env var, which is super bad.